src/cards/top-languages-card.js

```diff
@@ -160,18 +163,18 @@ const renderNormalLayout = (langs, width, totalLanguageSize) => {
  * @param {Lang[]} langs Array of programming languages.
  * @param {number} width Card width.
  * @param {number} totalLanguageSize Total size of all languages.
+ * @param {boolean} hideProgress Whether to hide progress bar.
  * @returns {string} Compact layout card SVG object.
  */
-const renderCompactLayout = (langs, width, totalLanguageSize) => {
+const renderCompactLayout = (langs, width, totalLanguageSize, hideProgress) => {
```

Comment:
You're using `hideProgress` in some places and `hide_progress` in other places. The GitHub mobile app sucks, so I'm not really sure if you let one equal the other somewhere, but either way, it would be better to have only `hide_progress` everywhere. We use `camelCase` for internal variables and `snake_case` for query parameters, but internal variables that represent query parameters also use snake case to avoid confusion. Please update your code to do the same!

Reply:
Okay, I checked the code out on my computer, and it seems fine!

Reply:
> Hi, thanks for the high quality PR! A few things before we merge:

Hello, and thank you.

> * No need to make changes to the docs in all languages! It's really amazing that you've taken the time to do that, but we're unfortunately in the process of migrating to an automated translation system, so we're not accepting any PRs regarding translated docs. I appreciate that you've taken the time to do that, and I apologize for the miscommunication.

Oh, okay, the translation system seems incredible.

> * I would think this is a valuable PR, but I'm not sure about the use case. Why would someone want to make the progress bars equal? It would be misleading to think that they spent equal time on all their languages. Hiding the percentages is a good feature, but again, I'm not sure about switching to compact mode. If the equal bars feature is removed, hiding percentages would make as much sense in the default layout as in the compact layout. Please consider my suggestion! @rickstaa please feel free to add your thoughts...

About the progress bar. In the `hire_progress=true` scenario, I only thought of it as juice, having the language colors instead of removing the bars. You're right; I can hide it.

About the default mode, I thought they would be so much alike, also, having a language per line without a progress bar looked a bit unnecessarily space-consuming.

I agree with anything you guys decide and will fix the issues.